include/experimental/__p1673_bits/blas3_triangular_matrix_matrix_solve.hpp

```diff
@@ -246,6 +246,128 @@ void trsm_lower_triangular_right_side(
 
 } // end anonymous namespace
 
+template<class ElementType_A,
+         extents<>::size_type numRows_A,
+         extents<>::size_type numCols_A,
+         class Layout_A,
+         class Accessor_A,
+         class Triangle,
+         class DiagonalStorage,
+         class ElementType_B,
+         extents<>::size_type numRows_B,
+         extents<>::size_type numCols_B,
+         class Layout_B,
+         class Accessor_B,
+         class ElementType_X,
+         extents<>::size_type numRows_X,
+         extents<>::size_type numCols_X,
+         class Layout_X,
+         class Accessor_X>
+void triangular_matrix_matrix_left_solve(
+  std::experimental::mdspan<ElementType_A, std::experimental::extents<numRows_A, numCols_A>, Layout_A, Accessor_A> A,
+  Triangle t,
```

Comment:
Would you consider commenting out `t` (as in `Triangle /* t */`) to prevent build warnings for `t` being unused?

Reply:
Sure would! How did you trigger those warnings? I didn't see them.

Reply:
@amklinv-nnl Greetings!  Sorry I missed your comment!  Adding `-Wall` to the list of warnings should help.